lib/stripe/stripe_object.rb

```diff
@@ -116,7 +116,7 @@ def as_json(*a)
 
     def to_hash
       maybe_to_hash = lambda do |value|
-        value.respond_to?(:to_hash) ? value.to_hash : value
+        value && value.respond_to?(:to_hash) ? value.to_hash : value
```

Comment:
Maybe change this to `!value.nil?` instead of just `value` to avoid changing the behavior for non-nil falsey values.

Reply:
Thanks :) I thought about that, the thing is in case of false assertion of this statement we end up just using the `value` and basically not call `to_hash` on it which _should_ be fine for any non-nil falsey values but let me know if you can think of any issues 🙌

Reply:
I always forget that `false` and `nil` are the only falsey values in Ruby. Should be fine then, as `FalseClass` does not have a `to_hash` method anyway!